spec/acceptance/mysql_server_spec.rb

```diff
@@ -6,7 +6,8 @@
       <<-MANIFEST
         class { 'mysql::server':
           manage_config_file      => 'true',
-          override_options        => { 'mysqld' => { 'key_buffer_size' => '32M' }},
+          #override_options        => { 'mysqld' => { 'key_buffer_size' => '32M' }},
```

Comment:
This is commented out, is this an oversight?

Reply:
yes, commented by mistake